pkg/cloud/aws/kinesis/shard_reader.go

```diff
@@ -463,5 +475,14 @@ func getShardReaderDefaultMetrics(stream Stream) metric.Data {
 			Unit:  metric.UnitCount,
 			Value: 0.0,
 		},
+		{
+			Priority:   metric.PriorityHigh,
+			MetricName: metricNameWaitDuration,
+			Dimensions: map[string]string{
+				"StreamName": string(stream),
+			},
+			Unit:  metric.UnitMillisecondsAverage,
```

Comment:
An average metric is problematic with a default - you get wrong results I think. Think about what happens if you have a single call to write 100 to that metric:

- you start with an initialization of 0
- you add the 100
- you now have a sum of 100 from 2 datapoints
- the result we write to cw is 50